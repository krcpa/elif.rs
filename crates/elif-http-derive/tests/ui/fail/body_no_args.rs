use elif_http_derive::{post, body};

#[post("/users")]
#[body]  // Missing body type argument
fn create_user() -> String {
    "created".to_string()
}