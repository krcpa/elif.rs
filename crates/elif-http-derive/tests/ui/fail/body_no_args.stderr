error: Body specification required. Hint: Use #[body(param_name: Type)] to specify the body parameter.
 --> tests/ui/fail/body_no_args.rs:4:1
  |
4 | #[body]  // Missing body type argument
  | ^^^^^^^
  |
  = note: this error originates in the attribute macro `body` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/fail/body_no_args.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/fail/body_no_args.rs`
